02_Development/Display/Tests/test_i2c_interface.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+import warnings
+
+from i2c_interface import I2CInterface, DeviceNotFoundError
+from rpi_check import is_on_raspberry_pi
+
+
+class TestI2CInterface(unittest.TestCase):
+
+    def setUp(self):
+        warnings.filterwarnings("ignore",
+                                message="I2C communication",
+                                category=UserWarning)
+        self._i2c = I2CInterface(0x70)
```

Comment:
`0x70` is def a magic number

Reply:
It's kind of meant to be a magic number I guess.  It just needs to be an I2C address with a device attached.  In this case it's the pressure sensor mux, but someone else that want's to run this test might have a device on 0x33 or any other.